src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -103,4 +104,20 @@ public function testObjectToPopulateWithProxy()
 
         $this->assertSame('bar', $proxyDummy->getFoo());
     }
+
+    public function testObjectWithStaticConstructor()
+    {
+        $normalizer = $this->getMockBuilder(ObjectNormalizer::class)
```

Comment:
Sorry, what I mean is create a custom normalizer to be used in the test that properly implements `getConstructor()`.